Kitodo-API/src/main/java/org/kitodo/api/validation/longtermpreservation/LtpValidationCondition.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.validation.longtermpreservation;
+
+import java.util.List;
+
+/**
+ * An immutable, no-setters beans implementing the validation condition interface.
+ */
+public class LtpValidationCondition implements LtpValidationConditionInterface {
+
+    private String property;
+    private LtpValidationConditionOperation operation;
+    private List<String> values;
+    private LtpValidationConditionSeverity severity;
+
+    /**
+     * Create a new (immutable, no-setters) validation condition.
+     * 
+     * @param property the property that is checked
+     * @param operation the operation a property is checked against
+     * @param values the values that a property is checked against
+     * @param severity the failure severity of this condition
+     */
+    public LtpValidationCondition(
+        String property, 
+        LtpValidationConditionOperation operation, 
+        List<String> values, 
+        LtpValidationConditionSeverity severity
+    ) {
+        this.property = property;
+        this.operation = operation;
+        this.values = values;
+        this.severity = severity;
+    }
+
+    /**
+     * Return the property of the file that is checked (e.g., ImageWidth, ColorSpace, etc.)
```

Comment:
Question: The property is a string. Is there a list somewhere? At least in the documentation/wiki, or something like that. We should have such a list so that we can check what we're testing here.

No need to change anything here.